providers/implementations/rands/seeding/rand_cpu_x86.c

```diff
@@ -52,11 +66,42 @@ size_t prov_acquire_entropy_from_cpu(RAND_POOL *pool)
             } else {
                 rand_pool_add_end(pool, 0, 0);
             }
+#endif
         }
     }
 
     return rand_pool_entropy_available(pool);
 }
+
+#if defined(OPENSSL_SYSNAME_TANDEM) && defined(_TNS_X_TARGET)
+/* Obtain random bytes from the x86 hardware random function in 64 bit chunks */
+static size_t get_hardware_random_value(unsigned char *buf, size_t len) {
+    size_t bytes_remaining = len;
+    while (bytes_remaining > 0) {
```

Comment:
Coding style put a blank line between declarations and code.  Here and elsewhere.

Reply:
Corrected - I hope.